berkeley-function-call-leaderboard/openfunctions_evaluation.py

```diff
@@ -55,6 +37,9 @@ def build_handler(model_name, temperature, top_p, max_tokens):
 def sort_key(entry):
     parts = entry["id"].rsplit("_", 1)
     test_category, index = parts[0], parts[1]
+    # index might be nested like 10-4-2. We only want the first part as that's the universal index.
+    if "-" in index:
+        index = index.split("-")[0]
```

Comment:
I have a nit, add comments on the id conversion difference b/t v1 and v2_live. Add an example and allow users to understand what each number represents. And also "universal index" might not be immediately clear without  context of how indexing works.

Reply:
addressed